metric-anomaly-data-model/src/main/java/org/hypertrace/alert/engine/metric/anomaly/datamodel/queue/KafkaQueueAlertTaskProducer.java

```diff
@@ -0,0 +1,60 @@
+package org.hypertrace.alert.engine.metric.anomaly.datamodel.queue;
+
+import com.typesafe.config.Config;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import jdk.jshell.spi.ExecutionControl.NotImplementedException;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.hypertrace.alert.engine.metric.anomaly.datamodel.AlertTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KafkaQueueAlertTaskProducer implements Queue<AlertTask> {
```

Comment:
should the task producer also be generic?

`class KafkaQueueTaskProducer<T> implements Queue<T>`

Reply:
Hey, I am moving out data-model changes to another PR to unblock both of us. Secondly,  I will make the queue generic enough as follow-up PR. So, we will just start with plain producer and consumer for now. 
Created a new PR (https://github.com/hypertrace/hypertrace-alert-engine/pull/9) as discussed.